android/app/src/main/res/values-hu/strings.xml

```diff
@@ -714,6 +714,24 @@
 	<string name="osm_wiki_about_url">https://wiki.openstreetmap.org/wiki/Hu:Névjegy</string>
 	<!-- Tip #00 -->
 	<string name="app_tip_00">Köszönjük, hogy használja a közösség által készített térképeinket!</string>
+	<!-- App tip #01 -->
+	<string name="app_tip_01">Adományaival és támogatásával létrehozhatjuk a világ legjobb térképes alkalmazását!</string>
+	<!-- App tip #02 -->
+	<string name="app_tip_02">Tetszik az alkalmazásunk? Kérjük, adományozzon a fejlesztés támogatására! Még nem tetszik? Kérjük, jelezze felénk, és mi megoldjuk!</string>
+	<!-- App tip #03 -->
+	<string name="app_tip_03">Ha ismer egy szoftverfejlesztőt, megkérheti őket egy olyan funkció megvalósítására, amelyre szüksége van.</string>
+	<!-- App tip #04 -->
+	<string name="app_tip_04">Tudja, hogy a térkép bármely pontjára hosszan koppintva kiválaszthatja azt?</string>
+	<!-- App tip #05 -->
+	<string name="app_tip_05">Tudja, hogy kiválaszthatja pozícióját a térképen?</string>
+	<!-- App tip #06 -->
+	<string name="app_tip_06">Segíthet alkalmazásunkat lefordítani az Ön nyelvére.</string>
```

Comment:
Put a "Ön is" to the start before "segíthet" and remove "Ön" at the second part of the sentence

Reply:
Please confirm: `Ön is segíthet alkalmazásunkat lefordítani az nyelvére.`

Reply:
"A nyelvére" is the correct form

Reply:
Modified to: `Ön is segíthet alkalmazásunkat lefordítani a nyelvére.`